fn_phish_tank/setup.py

```diff
@@ -0,0 +1,38 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+setup(
+    name='fn_phish_tank',
+    version='1.0.0',
+    license='MIT',
+    author='Resilient Labs',
+    author_email='resil.labs@gmail.com',
+    url='https://github.com/ibmresilient/resilient-community-apps',
+    description="Resilient Circuits Components for 'fn_phish_tank'",
+    long_description="""Check URL's against PhishTanks(https://www.phishtank.com/) Data base to see URL 
```

Comment:
For long_description, consider taking a sentence from the README and putting it here. I can see that is sort of whats happening here but instead, directly copy paste line 15 in the README and put it here.

Reply:
ok